src/app/components/event-page/event-page.component.html

```diff
@@ -1,84 +1,96 @@
-                    <div class="event-location-details">
-                        <div>{{event.eventLocation.street}} {{event.eventLocation.address}}</div>
-                        <div>{{event.eventLocation.city}}, {{event.eventLocation.province}}</div>
-                        <div><a href={{getMapLink()}}>View Map</a></div>
+        <div class="bottom-div">
+            <div class="event-description">
+                <!-- <div class="event-descripton-text">{{event.description}}</div> -->
+                <div class="event-descripton-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</div>
+                <mat-chip-list 
+                   class="tags-list"   
+                    cdkDropList 
+                    cdkDropListOrientation="horizontal">
+                    <mat-chip
+                        class="tag"
+                        cdkDrag
+                        *ngFor="let tag of tagsList">
+                            {{tag}}
+                    </mat-chip>
+                </mat-chip-list>
+            </div>
+            <div class="event-details">
+                <div class="event-details-text">
+                    <div class="location-text">
+                        <h2 class="location">Location:</h2>
+                        <div class="event-location-details">
+                            <div>{{event.eventLocation.street}} {{event.eventLocation.address}}</div>
+                            <div>{{event.eventLocation.city}}, {{event.eventLocation.province}}</div>
+                            <div><a href={{getMapLink()}}>View Map</a></div>
+                        </div>
                     </div>
-                </div>
-                <div>
-                    <div class="bold-element">Date & Time</div>
-                    <div class="starting-date">{{event.startingDate | date:'short'}} -</div>
-                    <div>{{event.endingDate | date:'short'}}</div>
-                </div>
-                <div>
-                    <div class="bold-element">Host</div>
-                    <div class="host-details">
-                        <div>Email: {{event.contactEmail}}</div>
-                        <div>Phone: {{event.contactPhone}}</div>
+                    <div>
+                        <h2 class="date">Date & Time:</h2>
+                        <div class="starting-date">{{event.startingDate | date:'dd.mm.yyyy, HH:ss'}} -</div>
```

Comment:
When you see something gets repetitive, create a constant for it :D. For example for the date format string.